docs/install.md

```diff
@@ -109,6 +109,29 @@ You can also temporarily switch to a specific version of Nextflow with the `NXF_
 NXF_VER=23.10.0 nextflow info
 ```
 
+## Use Nextflow with Seqera Platform
+
+To take your Nextflow usage to the next level, you can try
+[Seqera Platform](https://seqera.io/platform/).
+It's a tool created by the team behind Nextflow that's designed to monitor, manage,
+and optimise Nextflow workflows.
+It's free for small teams, and academic researchers can apply to get Seqera Cloud Pro for free.
+
+You can launch workflows directly from Seqera Platform, skipping the need to install Nextflow at all.
```

Comment:
```suggestion
You can launch workflows directly from Seqera Platform without installing Nextflow.
```

Reply:
Not sure about the language here. I mean, Nextflow is still installed _somewhere_, my point was that you don't need to faff around installing it locally.

Reply:
Updating it to "...installing Nextflow locally"

Reply:
The simplest solutions are often the best!